application/common/components/Emailer.php

```diff
@@ -402,13 +402,13 @@ public function shouldSendGetBackupCodesMessageTo($user)
 
     /**
      * @param User $user
+     * @param int $backupCodeCount - the number of backup codes left for the user (after a deletion)
      * @return bool
      */
-    public function shouldSendRefreshBackupCodesMessageTo($user)
+    public function shouldSendRefreshBackupCodesMessageTo($backupCodeCount)
```

Comment:
Since we're no longer passing a User to this function, you could remove the "...To" from the end of the function name (since it's not dealing with sending the message "to" anyone in particular). Looks like there's also an `@param` in the doc. comment that could be removed.

Reply:
Done